frontend/benefit/handler/src/components/newApplication/formContent/attachmentsList/AttachmentsList.tsx

```diff
@@ -0,0 +1,51 @@
+import { ATTACHMENT_TYPES } from 'benefit-shared/constants';
+import camelCase from 'lodash/camelCase';
+import * as React from 'react';
+import AttachmentsListBase from 'shared/components/attachments/AttachmentsList';
+import { BenefitAttachment } from 'shared/types/attachment';
+
+import { useAttachmentsList } from './useAttachmentsList';
+
+export type AttachmentsListProps = {
+  attachmentType: ATTACHMENT_TYPES;
+  attachments?: BenefitAttachment[];
+  handleQuietSave?: () => void;
+  required?: boolean;
+  as?: 'div' | 'li';
```

Comment:
`as` rings a bell. Is there something customized in this file or could this have been moved to benefit/shared from applicant side?